### PR TITLE
CastorBaseMonitor module removed, cleaning the rest Castor DQM modules

### PR DESCRIPTION
**Pull request by popovvp**
removed subFolder CastorEventProducts (Event Products plots moved to "Castor" subFolder
removed CastorBaseMonitor module
removed obsolete stuff from Castor modules
fixed bug in jet access

### CODE REVIEW

File: DQM/CastorMonitor/src/CastorRecHitMonitor.cc [hunk 1]
```diff
@@ -14,22 +14,24 @@
 //======================= Constructor ==============================//
 CastorRecHitMonitor::CastorRecHitMonitor(const edm::ParameterSet& ps)
 {
+ std::cout<<"CastorRecHitMonitor Constructor: "<<this<<std::endl;
```
Comment (dmitrijus) [inline DQM/CastorMonitor/src/CastorRecHitMonitor.cc]: Please remove this line.

**Comment by dmitrijus** [comment]
+1

**Comment by davidlange6** [comment]
@popovvp - please update the title of this PR with a meaningful description for the release notes. Thanks. (eg, a bug fix in what part of cmssw?)

**Comment by davidlange6** [comment]
+1

**Pull request merged**